### PR TITLE
Fix Timezone/Added Italian Language

### PR DESCRIPTION
**Pull request by MarioVillani**
Correct hours and minutes are shown using user's timezone.

### BEFORE - What was wrong? What was happening before this PR?

isoFormat did not take user's timezone into account.

### AFTER - What is happening after this PR?

Correct hours and minutes are shown.

### How did you achieve that, in technical terms?

Adding date_default_timezone_get() as second parameter for createFromTimeStamp() function.

### Is it a breaking change or non-breaking change?

Non-breaking

### How can we test the before & after?

Check if hours and minutes are shown correctly.

### CODE REVIEW

**Comment by MarioVillani** [comment]
EDIT: Added Italian Language strings.

File: src/app/Http/Controllers/BackupController.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ public function index()
                         'filePath'     => $file,
                         'fileName'     => $fileName,
                         'fileSize'     => round((int) $disk->size($file) / 1048576, 2),
-                        'lastModified' => Carbon::createFromTimeStamp($disk->lastModified($file))->isoFormat('DD MMMM YYYY, HH:mm'),
+                        'lastModified' => Carbon::createFromTimeStamp($disk->lastModified($file), date_default_timezone_get())->isoFormat('DD MMMM YYYY, HH:mm'),
```
Comment (pxpm) [inline src/app/Http/Controllers/BackupController.php:43]:
```suggestion
                        'lastModified' => Carbon::createFromTimeStamp($disk->lastModified($file), config('app.timezone'))->isoFormat('DD MMMM YYYY, HH:mm'),
```

**Review by pxpm — CHANGES_REQUESTED** [review]
Hey @MarioVillani thanks again for another PR. 

My only comment here is the same I did in your other PR: https://github.com/Laravel-Backpack/LogManager/pull/69#pullrequestreview-2135269422

Let me know what you think 🙏

**Comment by MarioVillani** [comment]
I was editing this after your suggestion for LogManager, you anticipated me :)
You may proceed with the merge if everything else is ok. Thanks!

**Pull request merged**